src/fu-util.c

```diff
@@ -3480,6 +3528,27 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 		}
 	}
 
+	/* any things we can fix? */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (!fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS) &&
+		    fwupd_security_attr_get_bios_attr_id(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_current_value(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_target_value(attr) != NULL) {
+			if (!fu_util_security_modify_bios_attr(priv, attr, error))
+				return FALSE;
+		}
+	}
+
+	/* reboot required? */
+	if (priv->completion_flags & FWUPD_DEVICE_FLAG_NEEDS_REBOOT) {
+		if (priv->no_reboot_check) {
+			g_debug("skipping reboot check");
+			return TRUE;
+		}
+		return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
```

Comment:
you might not get to prompt to upload results if you do it this way

Reply:
```suggestion
		if (!priv->no_reboot_check) {
			return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
		}
		g_debug("skipping reboot check");
```

Reply:
I've added a more comprehensive version, thanks.